File: Backend/src/models/Category.ts
import { Schema, model } from "mongoose";

export type CategoryDoc = {
  name: string;
};
const CategorySchema = new Schema(
  {
    name: {
      required: true,
      type: String,
    },
    desc: {
      required: true,
      type: String,
    },
  },
  { timestamps: true }
);

export const Category = model("Category", CategorySchema);

File: python/Backend/application/models/Category.py
from datetime import datetime
from application.models.db import *

class Category(Document):
    timestamps = DateTimeField(default=datetime.utcnow())
    name= StringField(required=True)
    desc=StringField(required=True)
    def to_json(self):
        return {
            "timestamps": self.timestamps,
            "name": self.name,
            "desc": self.desc,
        }